src/client/core/prevent-real-events.js

```diff
@@ -0,0 +1,71 @@
+import { utils, eventSandbox } from './deps/hammerhead';
+
+import { get, hasDimensions } from './utils/style';
+import { filter } from './utils/array';
+import { isShadowUIElement, isWindow, getParents } from './utils/dom';
+
+var browserUtils   = utils.browser;
+var listeners      = eventSandbox.listeners;
+var eventSimulator = eventSandbox.eventSimulator;
+
+const PREVENTED_EVENTS = [
+    'click', 'mousedown', 'mouseup', 'dblclick', 'contextmenu', 'mousemove', 'mouseover', 'mouseout',
+    'touchstart', 'touchmove', 'touchend', 'keydown', 'keypress', 'input', 'keyup', 'change', 'focus', 'blur',
+    'MSPointerDown', 'MSPointerMove', 'MSPointerOver', 'MSPointerOut', 'MSPointerUp', 'pointerdown',
+    'pointermove', 'pointerover', 'pointerout', 'pointerup'
+];
+
+const F12_KEY_CODE = 123;
+
+
+function checkDevToolsShortcut (e) {
+    // NOTE: Browser tools opening by F12, CTRL+SHIFT+<SYMBOL KEY> on PC or by OPTION(ALT)+CMD+<SYMBOL KEY> on Mac.
+    return e.shiftKey && e.ctrlKey || e.altKey && e.isMacPlatform || e.keyCode === F12_KEY_CODE;

```

Comment:
since we allow `ctrl+alt+<ANY KEY>` combination it's not only `DevToolsShortcut`, maybe it's `browserServiceShortcut` or `browserHotKey` or something else like this?